extensions/playerlist/common/client.mjs

```diff
@@ -0,0 +1,82 @@
+// Copyright 2021 HITCON Online Contributors
+// SPDX-License-Identifier: BSD-2-Clause
+
+import Overlay from '/static/sites/game-client/ui/overlay.mjs';
+import OverlayPosition from '/static/sites/game-client/ui/overlay-position.mjs';
+
+const PLAYERLIST_DIV = 'playerlist-overlay';
+
+class PlayerlistOverlay extends Overlay {
+  constructor(mainUI) {
+    const dom = document.getElementById(PLAYERLIST_DIV);
+    super(mainUI, dom);
+  }
+
+  // TODO(zeze-zeze): Expand on this class (jsdoc and implementation) and
+  // implement onPre/PostShow/Hide() as needed.
+
+  hasToolbarButton() {
+    this.toolbarButtonSrc = '/static/extensions/playerlist/playerlist.svg';
```

Comment:
I think Alan changed the way toolbar button works, but not sure if your PR is getting merged first or his is getting merged first.

Reply:
I think he just merged, you might have to update.